### ISSUE TITLE
Improve aria-labels

### ISSUE THREAD

**Issue by yasgreen93** [hex-digital/vue-accessible-datepicker#25]
- Add a better aria-label for the calendar icon button.
- Change the word "datepicker" to "calendar" - screenreader doesn't read it correctly.
